uspace/lib/c/generic/libc.c

```diff
@@ -80,6 +80,7 @@ void __libc_main(void *pcb_ptr)
 
 		main_fibril.tcb = tls_make_initial(__progsymbols.elfstart);
 	}
+	main_fibril.is_freeable = false;
```

Comment:
Where did this come from?

Reply:
some drive-by changes, as I was digging into fibril startup/exit because of hooks. Does the explicit assignment hurt something? I guess the zero (= false) value is there by default anyways... But it's a counterpart to this code: https://github.com/HelenOS/helenos/blob/0ae9e18465809a5520c51c76be2866b19c48bd0e/uspace/lib/c/generic/thread/fibril.c#L176

Reply:
Right, as far as I can tell the change does nothing, so I wondered what the rationale is. It doesn't seem like a helpful addition.

Reply:
I'm really not a fan of implicit zero-initialization as a valid value (other than "missing", i.e. NULLptr), but whatever